src/typechecker.erl

```diff
@@ -3510,7 +3510,7 @@ type_check_forms(Forms, Opts) ->
     StopOnFirstError = proplists:get_bool(stop_on_first_error, Opts),
     CrashOnError = proplists:get_bool(crash_on_error, Opts),
     File = proplists:get_value(print_file, Opts),
-    NoReportErrors = proplists:get_bool(no_report_errors, Opts),
+    NoReportErrors = not(proplists:get_bool(return_errors, Opts)),
```

Comment:
This is double negation, isn't it? It's the same as ReportErrors = ReturnErrors.

Reply:
Haha, yes you're right!